src/components/solvers/Graph/GraphArea.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Center, Container, Select, Spacer } from "@chakra-ui/react";
+import React, { useEffect, useRef, useState } from "react";
+import G6, { Graph } from '@antv/g6';
+
+interface GraphAreaProps {
+    graphData: any;
```

Comment:
Please avoid using `any`. I think the `GraphData` interface from the graph library should work here